documentation/src/components/code-example.ts

```diff
@@ -50,7 +51,22 @@ export class DarkCode extends Code {
 @customElement('light-code')
 export class LightCode extends Code {
     public static get styles(): CSSResultArray {
-        return [LightThemeStyles];
+        return [
+            LightThemeStyles,
+            css`
```

Comment:
Override code highlighting for accessible color application. The implementation is a little naive in the case that we ever leverage the `<dark-code>` element in the dark(est) themes.